scout4neo/scout4neo.go

```diff
@@ -0,0 +1,49 @@
+package scout4neo
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/project-regista/scout/configuration"
+
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+)
+
+func StoreData(statement []string) {
+
+	configuration.LoadConfig()
```

Comment:
We only need to load the configuration once during the program execution. You could do that in the main function or in the first function that is called and just use the viper.Get method to get the value that you need anywhere.